src/data/personalLinks.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactComponent as LinkedInIcon } from "../assets/linkedin.svg";
+import { ReactComponent as GitHubIcon } from "../assets/github.svg";
+import { ReactComponent as CvIcon } from "../assets/cv.svg";
+
+export interface PersonalLinksInfo {
+  icon: React.FunctionComponent<
```

Comment:
is it nessesary to use this complex icon type? Maybe ReactNode would be enough?

Reply:
I'm affraid it is. When type of icon is set to ReactNode TS throws an error.